crates/hyperdrive-math/src/long/close.rs

```diff
@@ -115,4 +159,52 @@ mod tests {
         assert!(result.is_err());
         Ok(())
     }
+
+    // Tests market valuation against yield space valuation when closing a long
+    // with the minimum transaction amount.
+    #[tokio::test]
+    async fn test_calculate_value_long() -> Result<()> {
+        let tolerance = int256!(1e15);
+
+        // Fuzz the spot valuation and yield space valuation against each other.
+        let mut rng = thread_rng();
+        for _ in 0..*FAST_FUZZ_RUNS {
+            let state = rng.gen::<State>();
+            let bond_amount = state.minimum_transaction_amount();
+            let maturity_time = state.position_duration();
+            let current_time = rng.gen_range(fixed!(0)..=maturity_time);
+
+            // Ensure curve_fee is smaller than spot_price to avoid overflows
+            // on the yield space valuation
+            if state.curve_fee() > state.calculate_spot_price()? {
+                continue;
+            }
+
+            let yield_space_valuation = panic::catch_unwind(|| {
+                state.calculate_close_long(bond_amount, maturity_time.into(), current_time.into())
+            })
+            .unwrap()
+            .unwrap();
+
+            let spot_valuation = state
+                .calculate_value_long(bond_amount, maturity_time.into(), current_time.into())
+                .unwrap()
```

Comment:
You can use the `?` syntax here instead of `unwrap()`, since the test itself returns a `Result<()>`. I don't _think_ there would actually be a difference here, fwiw, it's just for consistency.

Reply:
Fantastic! Updated.